### ISSUE TITLE
Error when closing files which have been moved in multiple panels 

### ISSUE THREAD

**Issue by ChrisLMartin** [spyder-ide/spyder#7869]
## Description

**What steps will reproduce the problem?**

1. Have open file in two panels.
2. Move file location.
3. Close file in one panel and save to new location.
4. Attempt to close file in other panel.

**What is the expected output? What do you see instead?**
File closes. Received error.

**Please provide any additional information below**

  File "C:\Users\christopher.martin\AppData\Local\Continuum\anaconda3\lib\site-packages\spyder\plugins\editor.py", line 1364, in close_file_in_all_editorstacks
    index = editorstack.get_index_from_filename(filename)
  File "C:\Users\christopher.martin\AppData\Local\Continuum\anaconda3\lib\site-packages\spyder\widgets\editor.py", line 1384, in get_index_from_filename
    return filenames.index(filename)
ValueError: 'C:\\Users\\christopher.martin\\Documents\\Python\\cal_data_processing\\cal_data_cement.py' is not in list

## Version and main components

* Spyder Version: 3.2.6 
* Python Version: 3.6.4
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.22.0 (OK)
numpy >=1.7      :  1.14.0 (OK)
sphinx >=0.6.6   :  1.6.6 (OK)
rope >=0.9.4     :  0.10.7 (OK)
jedi >=0.9.0     :  0.11.1 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.27.3 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.2.1 (OK)
pylint >=0.25    :  1.8.2 (OK)

```

**Comment by ccordoba12** [comment]
> Have open file in two panels.

What do you mean by this? Do you have two Spyder windows open at the same time?

> Move file location.

How did you do that? Please be more explicit.

> Close file in one panel and save to new location.

To understand what you mean by this, I need the answers to my previous two questions.

**Comment by ChrisLMartin** [comment]
> Have open file in two panels.

1 spider window, with the code editing portion split horizontally into two side by side panels.

> Move file location.

In Windows 10 File Explorer I dragged a *.py file that was open in Spyder into a subdirectory of its original location.

> Close file in one panel and save to new location.

I clicked the 'x' to close the *.py file that was open in both Spyder panels that I had moved. It alerted me that the file did not exist, so I saved it over the top of the new location that I had moved it to. This closed the file in only one of the two panels. I then clicked the 'x' on the other panel for the same file, which had not disappeared when I closed the same file on the first panel. This is when Spyder threw the error.
